frontend/app/src/people/widgetViews/BountyHeader.tsx

```diff
@@ -179,7 +179,7 @@ const FilterCount = styled.div<styledProps>`
 
 const EuiPopOverCheckboxLeft = styled.div<styledProps>`
   width: 147px;
-  height: 312px;
+  height: 372px;
```

Comment:
instead of just increasing the pixel size can you make it so the background is dynamic to the amount of stuff in the box incase we add more options in the future?

Reply:
Okay sure! Btw if you don't mind could you please give me any hint regarding how to implement it

Reply:
using `auto` might work instead of px values

Reply:
Okay! Thanks